packages/ubuntu_provision/lib/src/services/sound_service.dart

```diff
@@ -1,7 +1,18 @@
 import 'dart:io';
 
+import 'package:ubuntu_logger/ubuntu_logger.dart';
+
+final _log = Logger('sound_service');
+
+/// A service for playing sounds.
 class SoundService {
   Future<void> play(String id) async {
-    await Process.run('canberra-gtk-play', ['--id=$id']);
+    try {
+      await Process.run('canberra-gtk-play', ['--id=$id']);
+    } on ProcessException catch (e) {
+      _log.error(
+        'Error playing sound with id: $id (most likely missing canberra-gtk-play)',
```

Comment:
You need to either log `e` as well, or remove the variable

Reply:
Ah ofc!